tools/generators/prepare-initial-release/index.ts

```diff
@@ -236,9 +237,10 @@ async function stableRelease(tree: Tree, options: NormalizedSchema) {
   tree.rename(options.projectConfig.root, newPackage.root);
 
   return (_tree: Tree) => {
+    installPackagesTask(tree, true);
```

Comment:
install first so the new package name is properly registered within yarn workspaces